Tests/Extensions/DictionaryNSAttributedStringKeySpec.swift

```diff
@@ -0,0 +1,22 @@
+@testable import ELDebateFramework
+import Nimble
+import Quick
+
+class DictionaryNSAttributedStringKeySpec: QuickSpec {
```

Comment:
According to extension name changes please rename this class.

It should be renamed to `Dictionary_StringAnySpec`. 

Please don't forget to rename file as well.